Week2/alaa.js

```diff
@@ -0,0 +1,94 @@
+//1
+console.log('Hallo', 'wereld!'); // Dutch;
+console.log('hola', 'mundo!'); //spanish
+console.log('مرحبا', 'العالم!'); //Arabic
+//2
+console.log("I'm awesome"); // should use two quotations instead of one
+//3
+const x;
+console.log('the value of my variable x will be: undefined');
+x = 7; // initialize the value of x
+console.log('the value of my variable x will be 7');
+console.log(x);
+//4
+let y;
+y = 'hyf';
+console.log('the value of y will be :hyf');
+console.log(y);
+let y = 'code';
+console.log('the value of y will be :code');
+console.log(y);
+//5
+const z = 7.25;
+console.log(z);
+const a = Math.floor(z);
+console.log(a);
+const max = Math.max(z, a);
+console.log(max);//math.floor() rounds the number generated by math.random() down to the last integer. math round used to round a number to its nearest integer
+//6
+const myLibraries = [];
+console.log('the value of the the array is []');
+const myFavoriteAnimals = ['dogs', 'elephants', 'turtles', 'cats'];
+console.log(myFavoriteAnimals);
+const newLength = myFavoriteAnimals.push('baby pig');
+console.log(newLength);
+//7
+const len = myString.length;
+console.log(len);
```

Comment:
This is correct but you are missing the `myString` declaration. `let myString = "this is a test"`